modules/protocol/hybrid.cpp

```diff
@@ -22,7 +23,7 @@ class HybridProto : public IRCDProto
 		u->KillInternal(source, buf);
 	}
 
-  public:
+ public:
 	HybridProto(Module *creator) : IRCDProto(creator, "ircd-hybrid 8.2.23+")
```

Comment:
is hybrid 8.2.23 still the minimum version? If not please update it.

Reply:
It's still link compatible to 8.2.23 if that is what you mean. Older hybrid releases just don't have the new user-/channel modes.